test/nocks.js

```diff
@@ -40,14 +40,14 @@ function nockRegister(validator) {
       .reply(200);
 }
 
-function nockNumericProjectId(reply) {
+function nockProjectId(reply) {
   return nock('http://metadata.google.internal')
-    .get('/computeMetadata/v1/project/numeric-project-id')
+    .get('/computeMetadata/v1/project/project-id')
     .reply(reply);
```

Comment:
Add `.once` or .`.times(x)` can be helpful when looking at a nock instance 4-6 months down the line and try to understand its longevity.